<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMutualismBreakdownerSpatialPlotDialog</class>
 <widget class="QDialog" name="QtMutualismBreakdownerSpatialPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MutualismBreakdowner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #bbf, stop: 0.2  #bbf, stop: 0.5 #bfb, stop: 0.8 #fbb, stop: 1 #fbb);
  border: 1px solid black
}

QSpinBox,QDoubleSpinBox,QComboBox { 
  background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #bbb, stop: 1 #fff);
  border: 1px solid black
}

QPushButton
{
  color: #000;
  background-color: qradialgradient(cx: 0.5, cy: 0.5,fx: 0.5, fy: 0.5,radius: 0.5, stop: 0 #fff, stop: 1 #aaa);
  border: 1px solid black
}

QPlainTextEdit {
  font: 18px bold &quot;Courier&quot; courier monospace
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_left" native="true">
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_mid" native="true">
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_right" native="true">
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
